cmake/compiler/mwdt/target.cmake

```diff
@@ -0,0 +1,165 @@
+find_program(CMAKE_C_COMPILER ${CROSS_COMPILE}ccac PATH ${TOOLCHAIN_HOME} NO_DEFAULT_PATH)
+find_program(CMAKE_CXX_COMPILER ${CROSS_COMPILE}ccac PATH ${TOOLCHAIN_HOME} NO_DEFAULT_PATH)
+find_program(CMAKE_ASM_COMPILER ${CROSS_COMPILE}ccac PATH ${TOOLCHAIN_HOME} NO_DEFAULT_PATH)
+# The CMAKE_REQUIRED_FLAGS variable is used by check_c_compiler_flag()
+# (and other commands which end up calling check_c_source_compiles())
+# to add additional compiler flags used during checking. These flags
+# are unused during "real" builds of Zephyr source files linked into
+# the final executable.
+#
+# Appending onto any existing values lets users specify
+# toolchain-specific flags at generation time.
+list(APPEND CMAKE_REQUIRED_FLAGS
+  -c
+  -Hnosdata
+  -Hnolib
+  -Hnocrt
```

Comment:
-Hnocrt

isn't this what

-fno-freestanding

is?

Reply:
In Metaware, -Hnocrt is an option for linker, means donot link toolchain provided startup files.
It's similar to -nostartup

Reply:
This is OK then